### PR TITLE
Modernise the About Jenkins page

### PR DESCRIPTION
**Pull request by janfaracik**
### Old
image

### New
image

### Proposed changelog entries

* Modernise the "About Jenkins" page. Update the table and tab design to use normal cased text.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/hudson/AboutJenkins/index.jelly
Comment (timja) [inline core/src/main/resources/hudson/AboutJenkins/index.jelly:30]: can't you move this to the less file?

**Comment by daniel-beck** [comment]
I quite like how this looks 👍 It's an interesting take on the sidepanel, and splitting a long page up into tabs is also nice.

Wouldn't it be be good to start moving stuff into reusable Jelly tags to allow plugins to more easily align with core styles in a way that survived future style updates?  In this case, the table and "tabs"?

Other than that it might be good to revive https://plugins.jenkins.io/ui-samples-plugin/ so it can serve as a guide for plugin developers.

(I know I nag, but as a plugin maintainer I feel kinda lost thinking about how to adapt my stuff to make it look similar.)

File: war/src/main/js/section-to-tabs.js
Comment (timja) [inline war/src/main/js/section-to-tabs.js:2]: is `const` safe to use? supported by all browsers? or transpiled?

**Comment by timja** [comment]
Looks really nice, feels a bit weird not having a link to the Manage Jenkins page . Possibly because it's not included in the breadcrumbs for whatever reason?

image

**Comment by daniel-beck** [comment]
> Possibly because it's not included in the breadcrumbs for whatever reason?

It's the same everywhere, these are all top-level pages.

File: war/src/main/js/section-to-tabs.js
Comment (uhafner) [reply war/src/main/js/section-to-tabs.js:2]: I think yes. I use it in all JS files for a couple of years now.

File: war/src/main/js/section-to-tabs.js
Comment (uhafner) [inline war/src/main/js/section-to-tabs.js:48]: I wonder if we should include full Bootstrap library in core to avoid replicating things that someone else did (and supports) already.

**Review by uhafner — APPROVED** [review]
I must admit that I never opened that page before, but it looks really nice now.

**Review by NotMyFault — COMMENTED** [review]
Definitely an interesting approach with the panel on the left, but in my opinion it's a bit unneeded, considering can't do anything it with.
While the Jenkins version is present on the footer of any page, the panel itself takes up quite some screen space.
The table view itself looks nice though.

**Review by timja — APPROVED** [review]
I think we can go ahead with this. ATH has passed on it

---

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: core/src/main/resources/hudson/AboutJenkins/index.jelly
Comment (janfaracik) [reply core/src/main/resources/hudson/AboutJenkins/index.jelly:30]: Ah it's just so this page is styled, it'll be removed in a future PR when I get around to looking at some side panel updates :)

**Review by NotMyFault — CHANGES_REQUESTED** [review]
At higher zoom levels, currently looking at it with 200% at Chrome and Firefox, the Jenkins logo and text pop out to be centered at the screen:

**Comment by janfaracik** [comment]
> At higher zoom levels, currently looking at it with 200% at Chrome and Firefox, the Jenkins logo and text pop out to be centered at the screen:
> 

Fixed it :) 

image

**Review by NotMyFault — APPROVED** [review]
Acts fine by now.

**Review by oleg-nenashev — APPROVED** [review]
Looks great!

**Pull request merged**
